Dual stack configuration release note (#66996)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

When the Node Tuning Operator recognizes the performance conditions to activate the use of non-reserved CPUs, there is a several second delay while OVN-Kubernetes configures the CPU affinity alignment of OVS daemons running on the CPUs. During this window, if a `Guaranteed` QoS pod starts, it can experience a latency spike.
====

[id="ocp-4-14-dual-stack-configuration"]
==== Dual stack configuration for multiple IP addresses

> In previous releases of Whereabouts IPAM CNI, only one IP address could be assigned per network interface. 

Reviewer: ```suggestion
In previous releases of the Whereabouts IPAM CNI plugin, only one IP address could be assigned per network interface. 
```
Adding a little more context here for any readers who might not immediately know that Whereabouts is an IPAM CNI plugin.

> Now, Whereabouts IPAM CNI supports assignment of an arbitrary number of IP addresses in order to support dual-stack IPv4/IPv6 functionality.

Reviewer: ```suggestion
Now, Whereabouts supports the assignment of an arbitrary number of IP addresses to support dual-stack IPv4/IPv6 functionality.
```
- Since the first sentence states what Whereabouts is, I think we can get away with referring to it as simply "Whereabouts" in the second sentence.
- IBM Style advises using "to" instead of "in order to" in most cases. See the [Word Usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage) page and ctrl+F to find the entry for "in order to".

Author: Why not make it better? Thank you.
